src/Components/ColorPicker/ColorPicker.tsx

```diff
@@ -60,35 +57,26 @@ export const ColorPicker = forwardRef<ColorPickerSwatchRef, ColorPickerProps>(
       colors = influxColors,
       testID = 'color-picker',
       maintainInputFocus = false,
+      validationFunc = validateHexCode,
     },
     ref
   ) => {
-    const [errorMessage, setErrorMessage] = useState('')
+    const errorMessage = validationFunc(color)
```

Comment:
shouldn't this be more of a `validateHexCode(color) && validationFunc(color)` relationship? so that validating that a color matches a palette map somewhere doesnt mean you have to rewrite validateHexCode?

Reply:
I made `validationFunc` to be `validateHexCode` by default, but in case we ever want to support rgb values or named colors a different validation function can be passed in

Reply:
👍